src/PAModel/Schemas/adhoc/AppInsightsKey.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerPlatform.Formulas.Tools
+{
+    // Abstract out the InstrumentationKey (From AppInsights) of an app in a seprate json file (AppInsightsKey.json)
+    class AppInsightsKeyJson
+    {
+        public string InstrumentationKey { get; set; } // a guid
+
+        [JsonExtensionData]
+        public Dictionary<string, JsonElement> ExtensionData { get; set; }
```

Comment:
I think we can skip this - this json file is only in the extract case, so we control all the fields here.

Reply:
put another way, any extra fields here would *not* be added into the msapp.

Reply:
yea I was thinking the same, will remove it.